### PR TITLE
Filesystem read

### PR DESCRIPTION
**Pull request by SMartinScottLogic**

### CODE REVIEW

File: dumps/linux_nonpartioned.dump
Comment (SMartinScottLogic) [inline dumps/linux_nonpartioned.dump:1]: Remove this

File: dumps/linux_nonpartioned.dump.txt
Comment (SMartinScottLogic) [inline dumps/linux_nonpartioned.dump.txt:1]: Remove this

File: dumps/linux_partioned.dump
Comment (SMartinScottLogic) [inline dumps/linux_partioned.dump:1]: Remove this

File: dumps/linux_partioned.dump.txt
Comment (SMartinScottLogic) [inline dumps/linux_partioned.dump.txt:1]: Remove this

File: src/fat12_partition/setup.rs
Comment (SMartinScottLogic) [inline src/fat12_partition/setup.rs:1]: Convert to use `include_bytes!(...)`

**Comment by bobrippling** [comment]
Merged as part of #5

**Pull request closed without merging**